i18n/de/docusaurus-plugin-content-docs/current/help-center/your-account/account-settings.mdx

```diff
@@ -488,6 +488,37 @@ Abhängig davon in welchem Status sich das Chat-Backup befindet, haben Sie zwei
 
 In beiden Fällen, stellen Sie sicher, dass Sie die erstellte Passphrase sicher aufheben.
 
+## Chat-Einstellungen
+
+### Verhalten der Eingabetaste ändern
+
+In diesem Abschnitt finden Sie eine Anleitung, wie Sie das Verhalten der Eingabetaste verändern können.
+
+:::info
+Diese Einstellung lässt sich für jedes Gerät einzeln vornehmen und wird auf die Standardeinstellung zurückgesetzt, wenn du dich über die Profileinstellungen abmeldest oder das Gerät unter „Verbundene Geräte“ in den Einstellungen entfernst. Nach der Anmeldung musst du die Einstellung dann erneut vornehmen.
```

Comment:
Wir Duzen die Nutzer nicht.

Reply:
Change "Diese Einstellung lässt sich für jedes Gerät einzeln vornehmen und wird auf die Standardeinstellung zurückgesetzt, wenn du dich über die Profileinstellungen abmeldest oder das Gerät unter „Verbundene Geräte“ in den Einstellungen entfernst. Nach der Anmeldung musst du die Einstellung dann erneut vornehmen." to:
**"Diese Einstellung lässt sich für jedes Gerät einzeln vornehmen und wird auf die Standardeinstellung zurückgesetzt, wenn Sie sich über die Profileinstellungen abmelden oder das Gerät unter „Verbundene Geräte“ in den Einstellungen entfernen. Nach der Anmeldung müssen Sie die Einstellung dann erneut vornehmen."**

Reply:
Changed.